internal/install/command.go

```diff
@@ -63,7 +63,10 @@ var Command = &cobra.Command{
 				return
 			}
 
-			log.Fatalf("We encountered an error during the installation: %s. If this problem persists please visit the documentation and support page for additional help here: https://one.newrelic.com/-/06vjAeZLKjP", err)
+			fmt.Println()
+			fmt.Printf("We encountered an error during the installation: %s.\n\n", err)
```

Comment:
@sanderblue I think that whole phrasing should be reviewed (changed), I dont think we should say "error".
Check with Sam/Aaron, perhaps more something like "We couldn't complete this installation, details :%s" and send the user back to the UI.

Reply:
Agreed. I mentioned it to Sam. In general the output needs additional organization to make it friendlier to the users' eye.

Reply:
I assume you'll reach out to him to finalize those details? Let me know when done and I'll start testing.

Reply:
I'll confirm tomorrow.

Reply:
Closing the loop, we're changing this to a debug log since the general output points the customer back to finishing any incomplete installations. As far as I know, we only have one scenario where we run into this error and that's when the recipes are not compiled, so likely will only occur during local dev. I've accounted for both of these scenario 😎